### ISSUE TITLE
Add ability to print out environment and makefile variables

### ISSUE THREAD

**Issue by alexlin0** [nasa/trick#603]
If we add these lines to the top level sim makefile

```makefile
print-%:
        @echo '$*=$($*)'
```
This will let you print environment variables and makefile variables from the command line

e.g.

```
make print-TRICK_SFLAGS
make print-TRICK_CFLAGS
make print-TRICK_SYSTEM_CFLAGS
```

**Comment by iamthad** [comment]
Isn't this already in [Makefile.sim](https://github.com/nasa/trick/blob/master/share/trick/makefiles/Makefile.sim#L79)?

**Comment by alexlin0** [comment]
I thought I already put it in there.  I couldn't find it the other day.  I must have been in a different version... Thanks for point it out.  Yup, closing this issue.

**Comment by iamthad** [comment]
I just tried this on one of my sims and it didn't seem to work. Can you confirm that it does?

**Comment by alexlin0** [comment]
It should be deleted.

**Comment by spfennell** [comment]
Fixed in #623
